backend/src/main/java/com/benine/backend/video/MJPEGFrameHeader.java

```diff
@@ -0,0 +1,87 @@
+package com.benine.backend.video;
+
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created on 01-06-16.
+ */
+public class MJPEGFrameHeader {
+
+  private int contentlength;
+  private String boundary;
+  private String contenttype;
+
+  private static final String MJPEG_BOUNDARY = "--polycast";
```

Comment:
Oh I didn't properly understand the usage of it then. I would suggest changing it to "--benine" or "--start[something]" since this product is focussed on polycast but not for their use only.